resources/css/bem/user-level.less

```diff
@@ -6,7 +6,65 @@
   width: 50px;
   height: $width;
   font-size: @font-size--large;
-  background-image: url('~@images/backgrounds/level-hexagon.svg');
```

Comment:
put back the original svg except modify it

```svg
<svg xmlns="http://www.w3.org/2000/svg">
<defs>
  <clipPath id="clip" clipPathUnits="objectBoundingBox" transform="translate(3.35) scale(0.01)">
    <original path goes here/>
  </clipPath>
</defs>
</svg>
```

And the icon element

```css
&__icon {
  .full-size();
  clip-path: url('~@images/backgrounds/level-hexagon.svg#clip');
  background: linear-gradient(var(--bg));
}
```

Finally the base block modifiers (don't bother saving off typing `--tier`)

```scss
&--tier-<name> {
  --bg: @level-tier-<name>;
}
// putting the original variables in layout.less as css variables
// and using them inlined directly could also work
```

The icon element can be just an empty div.

Reply:
These changes should be applied now